src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function getMaxWidth($width, $quantity, $gap, $padding) {
```

Comment:
Consider adding a brief comment to describe the purpose of the getMaxWidth function.